pkg/provider/gke/gke.go

```diff
@@ -103,11 +89,25 @@ func (c *GKE) NewGKEClient(*kingpin.ParseContext) error {
 		c.Auth = string(auth)
 	}
 
+	// Create tempory file to store the credentials.
+	saFile, err := ioutil.TempFile("", "service-account")
+	if err != nil {
+		return err
```

Comment:
```suggestion
		return errors.Wrap(err,"create temp file")
```

To make it more clear which part of the code returned the error.

The same for the one below